ocs_ci/templates/workloads/couchbase/pull_secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: 711497-qe-test-pull-secret
+data:
+  .dockerconfigjson: eyJhdXRocyI6eyJyZWdpc3RyeS5yZWRoYXQuaW8iOnsiYXV0aCI6Ik56RXhORGszZkhGbExYUmxjM1E2WlhsS2FHSkhZMmxQYVVwVFZYcFZlRTFwU2prdVpYbEtlbVJYU1dsUGFVa3pXbTFLYlU1WFZtdE9NbEp0VFRKTk1FMVVVVEZaYWs1cVRWZEpNRnBYVlhsYVIwazBXV3BuZWsxVFNqa3VSazVHYVcxNFQwbEdNemwxU0RWb1pHeE1VbU5zU0hSZk1IZGZVVFpOUTE5eGRHTXhTemh5TlZkd1NuTlRPSEk0V21WdFZWVlRWRXBQUzNKTGQzTmtZMEZUUm5WVVVFWlpkbloyWXpCdE9VOW5lSE55VjFGdGIzSmxiMncwZDFSRFN6SnhZMmx1ZVZaMmRXdFpTVVZmVTE5Q2VHTnVRemhmU0ZkS2NXVnFRbVYyWDNSVFUwOVFiMWhQUlZoWU9WQmhNbVIxTVhwc1RqUkJNa2huYkhWcVZWcFljbGRqY21kSmFVMHRZVGQ0UzB4V2EzSlBNSEkwTldGU0xUUlZOR1F5V0hkcU4yaHZNVTl3Y1hWMFRqRnNSM28yTTJWb09VbG1TREJRT0hBMlYxQnZXSEJ4WHpKbmIwTkJYemRzZGs5S1prMVFSRVpzVEc5ak1GRktWRTFoUTJ0WWNWb3dVV3hUY0hBeFIwSm9TVnBzUmsxalUxbDRTVlEzZGpaelRIUmxURGhVUkZWTVIyUlFlV3MxUkhwd1gzZHVibFF0VEhoSmNrbzRVVTlHUld0cmRIVnVWamhSVWxGWFFWaHBRUzEzZEU4MVVGWjRNR1ZHUlV0V1pWWlhTbkp1Ums5dFJWcENZM05pYTBRd1QyWTJZV1JXV1dkRVZGVkhTVU5pVmxCUE9YQjRhaTFFVmxGNFF5MXBhbkl0TVdadmRuRnFWRmwwVm1Wak1VWXdTRXg2UVdrM1NrbHdNRkZrZUd0d1ptRnBRamhqVjBJMVVGaFlUMlp4ZDB0RE9XbHJTWEpGTTB0T1gwOTNhMGRYYUdReWQwOWlWMVZ2UmpobVJ6SkRTV2RwY0RSTFQzQXpSRlJvZWxKWVpIQnJWVGhUWlRGSllteHVPVFpGTTBGRlZuVlJXbTlOZG14Nk4zaHZaVk5TYTA1NVUyVlBaVTVQVlVsRGJuVlJabTlQVG0xa016aG1ORk5aVEVORlprZEZSV3BvZUVGSmIwZEtXV0ZSVjNrMWNVZEpSM2s0WlZNNFFuWkVXbEpIVW10Nk5HZG5Xa1pUV2pVeGVtNHhObEozYjFGMlJtaDVTMEZPUldkd2RXaFJMVVk1U2xGcFdESlpVMHhqY0c1SFJFRk1OVk50ZGkxeFZHcDBSR2hPYXpCZlNsOVJMVk40WWxOck5tMWxZUzAxTTBGTmIxRkhkR0pZWTBFMlRtYz0ifX19
```

Comment:
2 concerns here.
1) we expose your secret 
2) I doubt if using this secret will be useful when tests are run by other users?

Reply:
I have moved this secret to a data file that will not get checked in.